docs/source/datasets.rst

```diff
@@ -307,71 +307,62 @@ When retrieving a single dataset record, the following version-related fields ar
-
-On further changes to ``access_type`` from permit to non-permit, all existing applications are closed and users must request a new permission for downloading. Furthermore, the ``catalogue-item`` related to the dataset will also be disabled and cannot be used further.
-
-.. important:: After changing the access type of a dataset to non-permit, REMS will not be used anymore for this dataset.
+        # ... other fields
+        "access_rights": {
+            # ... other access rights
+            "access_type": {
+                "identifier": "http://uri.suomi.fi/codelist/fairdata/access_type/code/permit"
+            }
+        }
+        # ... other fields
 
-When changing the ``license`` of the dataset, similar closing and disabling operations happen in REMS. All the applications are closed and the corresponding ``catalogue-item`` is disabled. But in addition to these, a new ``catalogue-item`` will be created to match the changed license. This new ``catalogue-item`` will be used for further applications for this dataset.
+    More information about updating a dataset can be found in :ref:`Update examples<rst-dataset-examples-update>`.
 
-.. important:: After license change operation, REMS will still be used to manage the file downloads for this dataset but all the previous applications are closed and new applications are needed to get download access to the files of the dataset.
 
+    **Changing license**
 
-**Access granter**
+    License is required property for those datasets that are managed by REMS. This license is what a downloading user must agree to. If there are multiple licenses described in dataset, REMS only considers the first one. So changing the license in REMS is changing the first license in the dataset. Example of defining a license:
 
-``access_granter`` parameter is needed in requests that changes the access_type to permit. This will define the user, who will decide whether to give download access or not. The user cannot be changed after it is defined. The parameter value is base64 encoded JSON string with attributes of ``userid``, ``name`` and ``email`` containing fairdata userid, name and email of the access granter respectively. Example:
+    .. code-block:: python
 
-.. code-block:: python
+        # ... other fields
+        "access_rights": {
+            # ... other access rights
+            "license": [
+                {
+                "identifier": "http://uri.suomi.fi/codelist/fairdata/license/code/CC0-1.0"
+                }
+            ]
+        }
+        # ... other fields
 
-    from base64 import urlsafe_b64encode
-    import json
+    Please refer to :ref:`Update examples<rst-dataset-examples-update>` for more information about update process.
 
-    access_granter = {
-        "userid": "jodoe1",
-        "name": "John Doe",
-        "email": "john.doe@example.com"
-    }
```

Comment:
Is there now any example anywhere what the access_granter object should look like? This would be useful info for other service devs.